### ISSUE TITLE
Improve page load times for completed shopping-cart sample app

### ISSUE THREAD

**Issue by michaelpnash** [lightbend/cloudstate-samples#258]
When finished deploying the shopping cart, the time to load the UI page is around 10 seconds. This is sufficiently slow to count as broken, and should be optimized in whatever way it needs (cache?)

A reasonable goal (IMO) is under 2 seconds

**Comment by katsutoxin** [comment]
@michaelpnash  Do you mean this shopping cart sample app?  https://github.com/lightbend/cloudstate-samples/tree/master/shopping-cart

**Comment by michaelpnash** [comment]
Yes. We apparently have a 10+ second load time for the primary page of the finished app - that gives a bad impression. Corey probably has some thoughts on the reasons.

**Comment by katsutoxin** [comment]
I did a quick test against https://danny-project-1.us-east1.apps.lbcs.dev/pages/index.html


It takes around 5 seconds to load the page.

The main reason is that we don't minify the file bundle.js, its size is almost 6MB, and it takes around 3 seconds to download it. 

<img width="1267" alt="Screen Shot 2020-06-01 at 11 09 36 AM" src="https://user-images.githubusercontent.com/11674700/83439840-bf9a5180-a3f8-11ea-8dcd-b8894e61166f.png">

**Comment by hseeberger** [comment]
Good catch!

**Comment by katsutoxin** [comment]
After I minify the bundle.js, the loading performance is much better. The bottleneck bundle.js now takes less then 1 second to load. The total loading time is half of the original  

<img width="1275" alt="Screen Shot 2020-06-01 at 9 14 29 PM" src="https://user-images.githubusercontent.com/11674700/83479315-192c6b80-a44d-11ea-9186-3a93c324ece5.png">

**Comment by michaelpnash** [comment]
@katsutoxin Great improvement here, many thanks!
